Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM centos:centos7
```

Comment:
add copyright

Reply:
Done.

Reply:
Wait, copyright or license?

Reply:
I'm assuming this was for license which has been added.